### ISSUE TITLE
[DDL][Board Appeals] Closed Appeal

### ISSUE THREAD

**Issue by jacobworrell** [department-of-veterans-affairs/va.gov-team#55273]
## Value Statement
**_As a_** veteran waiting on my board of appeals decision
**_I want to_** be able to find my appeal decision letter once my appeal is closed
**_So that_** I'm able to download it and take whatever action necessary to pursue a favorable outcome for me.

---
## Background Context
- Just as we use a closed claim to link users to the decision letter listview, we should use the same approach for a closed appeal. 

## Acceptance Criteria
- [ ]  Closed board appeals provide a link to the DDL List View
- [ ]  Language associated with link makes it clear that the letter is ready now that the appeal is closed.
- [ ]  Passes accessibility standards.

## Designs and Build Notes
- Add here

## Out of scope
- Add here

## Open questions
- Add here

## Tasks
- [ ] _Task_
- [ ] _Task_
- [ ] _Task_

## Definition of Ready 

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Reviewed and approved by product and/or design

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by rubensun** [comment]
Content considerations:
- Veterans are concerned about the status of the claim as pertaining to the decision of the claim.
- Some users interpret the term **closed** with **denied**.
- It's best to frontload the decision in the message over the claim status.

Consider this language for the message.
> ### The determination of your claim is now available.
> The VA made a determination on your claim on September 21, 2022. The claim has now been closed.
> To view the determination, download your decision letter through the link below. In this section, you will also be able to get other letters > related to your claim.
>
> A copy of the decision letter has also been sent by mail. It should arrive within 10 days after the date we closed your claim, but it may take longer.
>
> [Get your claim letters](url)
